winlogbeat/eventlog/eventlog.go

```diff
@@ -82,39 +84,63 @@ type Record struct {
 func (e Record) ToEvent() beat.Event {
 	m := common.MapStr{
 		"type":          e.API,
-		"log_name":      e.Channel,
-		"source_name":   e.Provider.Name,
-		"computer_name": e.Computer,
 		"record_number": strconv.FormatUint(e.RecordID, 10),
-		"event_id":      e.EventIdentifier.ID,
 	}
 
+	m.Put("event.kind", "event")
+	m.Put("event.dataset", fmt.Sprintf("%v.%v", e.API, strings.ToLower(e.Channel)))
+	m.Put("log_name", e.Channel)
+	m.Put("source_name", e.Provider.Name)
+  // Why not keep the Windows naming for those?
+	m.Put("channel", e.Channel)
+	m.Put("provider_name", e.Provider.Name)
+	m.Put("event.code", e.EventIdentifier.ID)
+	addOptional(m, "event.action", e.Task)
```

Comment:
Sample values:

- Logon
- Logoff
- Special Logon
- Credential Validation

However this field was set less than 50% of the time, in my playing around...